packages/server/modules/core/services/streams/management.ts

```diff
@@ -88,12 +90,23 @@ export async function createStreamReturnRecord(
   // Invite contributors?
   if (!isProjectCreateInput(params) && params.withContributors?.length) {
     // TODO: should be injected in the resolver
-    await inviteUsersToStreamFactory({
+    await inviteUsersToProjectFactory({
       createAndSendInvite: createAndSendInviteFactory({
-        findResource: findResourceFactory(),
         findUserByTarget: findUserByTargetFactory(),
-        insertInviteAndDeleteOld: insertInviteAndDeleteOldFactory({ db })
-      })
+        insertInviteAndDeleteOld: insertInviteAndDeleteOldFactory({ db }),
+        collectAndValidateResourceTargets: collectAndValidateCoreTargetsFactory({
+          getStream
+        }),
+        buildInviteEmailContents: buildCoreInviteEmailContentsFactory({
+          getStream
+        }),
+        emitServerInvitesEvent: ({ eventName, payload }) =>
```

Comment:
I've just been passing `emit` directly in cases like this. Something like:

```
const { emit } = getEventBus()
emitServerInvitesEvent: emit
```

I think this "does everything" event bus was created after we all discussed factory functions/domain types. So the naming here feels a bit silly to me. (`emitServerInvitesEventButReallyAnythingYouWant`)

So this is kind of an ignorable style nit but I had some thoughts while writing it out. Maybe we reconsider how these get created/injected.